outermysite/polls/imdbExactApi.py

```diff
@@ -0,0 +1,92 @@
+import requests
+import sys
+import traceback
+from bs4 import BeautifulSoup
+
+def getMovieResults(Id):
+    
+    url = "http://www.imdb.com/title/"+Id+"/?ref_=fn_al_tt_1"
+    
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        print "Unable to connect"
+        raise SystemExit("Slow internet connection")
+    
+    soup = BeautifulSoup(r.content, "html.parser")
+    
+    movieDetails = soup.find("div", {"class" : "plot_summary"})
+    
+    jsonArray = []
+    
+    if movieDetails==None:
+        print "No results found"
+        jsonObject={}
+        jsonObject['Error']="No results found"
+        return jsonObject
+    
+    description = movieDetails.find("div", {"class" : "summary_text"})
+    
```

Comment:
Probably too much spacing if you are adding a blank line after every line of code.